include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -2,15 +2,73 @@
 
 #include<iostream>
 #include<pjsua2.hpp>
+#include<unordered_map>
+#include <string>
 
 #include"Softphone.hpp"
+#include"SoftphoneArguments.hpp"
 
 class SoftphoneManager
 {
 public:
-    SoftphoneManager();
-    ~SoftphoneManager();
+    SoftphoneManager(int number_of_softphones, int port, std::string domain):
```

Comment:
Remove `number_of_softphones` from here as it is not used

Reply:
done